app/src/app/components/header/Header.tsx

```diff
@@ -4,37 +4,36 @@ import { AccountHeaderDropdown } from "./AccountHeaderDropdown";
 
 import { useUser } from "../providers/UserProvider";
 import { ThemeToggleButton } from "./ThemeToggleButton";
-import { cn } from "../../../lib/cn";
-import { buttonVariants } from "../Base/Button";
-import { NavMenuMobile } from "./NavMenuMobile";
-import { LeftNav } from "./LeftNav";
-import { hasUserManagePermission } from "../../../lib/auth/hasPermission";
+import { NavMenu } from "./NavMenu";
+import { useBrandingContext } from "../providers/BrandingProvider";
 
 export const Header = () => {
   const { user } = useUser();
+  const { logoConfig, logoConfigIsLoaded, brandName } = useBrandingContext();
+  const logoLinkDestination = logoConfig?.linkDestination || "/";
 
   return (
     <header>
-      <div data-testid="header" className="flex-col">
-        <div className="border-b shadow-sm dark:shadow-accent">
-          <div className="flex h-20 items-center px-4">
-            <NavLink to="/">
-              <div className="text-xl font-extrabold flex gap-1 items-center">
-                <PackageOpen />
-                Packit
-              </div>
-            </NavLink>
-            {user && <NavMenuMobile user={user} />}
-            {user && <LeftNav className="mx-6 hidden md:flex" user={user} />}
-            <div className="ml-auto flex items-center space-x-4">
-              {/* <NavigationLink to="/accessibility" className="mx-6 hidden md:flex">
-                Accessibility
-              </NavigationLink> */}
-              {hasUserManagePermission(user?.authorities) && (
-                <NavLink to="/manage-roles" className={cn(buttonVariants({ variant: "ghost" }), "justify-start")}>
-                  Manage Access
+      <div data-testid="header">
+        <div className="flex items-center h-20 border-b shadow-sm border-accent">
+          {logoConfigIsLoaded && brandName && (
```

Comment:
`brandName` should always be set shouldn't it, since it comes from the document title? Or does it get snarled up in some async via the provider?
I suppose someone could very foolishly set their custom brand name to be an empty string. They will have a bad time anyway..

Reply:
Removed check